packages/playground/src/views/not_yet_implemented.vue

```diff
@@ -1,9 +1,43 @@
 <template>
-  <div>not_yet_implemented</div>
+  <img class="centered-img" src="images/404.png" />
+  <p class="text1">Oops. The page you were looking for doesn't exist.</p>
+  <p class="text2">You may have mistyped address or the page may have been moved</p>
+  <div style="text-align: center">
+    <a href="/" class="link-text"> Take me back to the home page</a>
+  </div>
 </template>
 
 <script lang="ts">
 export default {
   name: "NotYetImplemented",
 };
 </script>
+
+<style>
+.centered-img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 55%;
+}
```

Comment:
```suggestion
.centered-img {
  display: block;
  margin-left: auto;
  margin-right: auto;
  width: 100%;
  max-width: 700px; // or something else
}
```

Please use width:100% and max-width for images to avoid the over shrinking of the images in small screens

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/31689104/7ef45a17-be35-4514-929e-6b125df40a9f)